packages/cli/src/models/network/NetworkBaseController.js

```diff
@@ -344,4 +345,25 @@ export default class NetworkBaseController {
     await this.project.freeze()
     this.networkFile.frozen = true
   }
+
+  async _saveOwner() {
+    const app = this.networkFile.app;
+    if (app.owner == null) {
+      const owner = this.txParams.from || await getAccount(0);
+      app.owner = owner;
+      this.networkFile.app = app;
+    }
+    const appPackage = this.networkFile.package;
+    if (appPackage.owner == null) {
+      const owner = this.txParams.from || await getAccount(0);
+      appPackage.owner = owner;
+      this.networkFile.package = appPackage;
+    }
+    const provider = this.networkFile.provider;
+    if (provider.owner == null) {
+      const owner = this.txParams.from || await getAccount(0);
+      provider.owner = owner;
+      this.networkFile.provider = provider;
+    }
+  }
```

Comment:
I think it is more correct to save this info when we create these contracts. For example, if you already have deployed these contracts and you run the push command, you will be storing the current sender address as the owner of those contracts you deployed before, which may not be right. 
Additionally, you could look for the owner only once with `const owner = this.txParams.from || await getAccount(0);`

Reply:
It would be nice to track the owner when creating with `zos create` as well, for those not publishing.